### PR TITLE
fixed logback files

### PR DESCRIPTION
**Pull request by dartpopikyardo**

### CODE REVIEW

File: java-server/api-gateway-service/src/main/resources/logback.xml [hunk 1]
```diff
@@ -14,6 +14,6 @@
     </root>
     <logger name="org.springframework" level='info'>
     </logger>
-    <logger name="io.eventuate.client.java" level='info'>
```
Comment (cer) [inline java-server/api-gateway-service/src/main/resources/logback.xml:17]: `io.eventuate.client.java` can probably stay

File: java-server/api-gateway-service/src/main/resources/logback.xml [hunk 2]
```diff
@@ -14,6 +14,6 @@
...
+    <logger name="io.eventuate.activity" level='info'>
```
Comment (cer) [inline java-server/api-gateway-service/src/main/resources/logback.xml]: `io.eventuate.activity` needs to be debug level.

**Review by cer — CHANGES_REQUESTED** [review]

File: java-server/api-gateway-service/src/main/resources/logback.xml (hunk 2 above)
Comment (dartpopikyardo) [reply java-server/api-gateway-service/src/main/resources/logback.xml]: fixed

File: java-server/api-gateway-service/src/main/resources/logback.xml (hunk 1 above)
Comment (dartpopikyardo) [reply java-server/api-gateway-service/src/main/resources/logback.xml:17]: fixed

**Pull request merged**